apps/public-docsite-v9/.storybook/main.utils.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * This file contains utils for main.js to mitigate diffs when migration generator is being invoked in batch (via --all flag)
+ * Code in this module is supposed to run only against node js env (webpack) - thus it uses commonjs modules
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+function getVnextStories() {
```

Comment:
what about react-components ? we will have duplicate code in both places ?

Reply:
If desired, I can remove dup code from react-components....that just seemed more churn in a single PR